src/content/docs/synthetics/synthetic-monitoring/using-monitors/device-emulation.mdx

```diff
@@ -0,0 +1,62 @@
+---
+title: Device emulation
+tags:
+  - Synthetics
+  - Synthetic monitoring
+  - Using monitors
+translate:
+  - jp
+metaDescription: Emulating mobile or tablet devices in browser based synthetic monitors
+redirects:
+---
+
+Device emulation is an optional setting available for simple and scripted browser monitors. This setting simulates the viewport and capabilities of a mobile or tablet device using the emulation capabilities of Chrome DevTools. 
+
+Device emulation allows you to test a browser based application using a simulated generic mobile or tablet device. This setting does not change the network, user agent, or device type being used to execute your synthetic monitor. Device emulation does not allow you to install and test mobile applications.
```

Comment:
@bpeckNR what do you think of a small revision like this:

Device emulation allows you to test a browser-based application using a simulated generic mobile or tablet device. Keep the following in mind:

* This setting does not change the network, user agent, or device type being used to execute your synthetic monitor. 
* Device emulation does not allow you to install and test mobile applications.

Reply:
Looks perfect. My next commit includes this.